TetrisStyleGame/Board.py

```diff
@@ -26,12 +26,7 @@ def __init__(self, rows=20, columns=10):
     def _find_full_rows(self):
         full_rows = []
         for r, row in enumerate(self.rows):
-            count = 0
-            for cell in row:
-                if cell.color != EMPTY:
-                    count += 1
-
-            if count == len(row):
+            if 0 == len([cell for cell in row if cell.color == EMPTY]):
```

Comment:
This might need a comment to the effect of 'check for rows with 0 empty cells' 

What's happening in this code is not immediately clear